modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,18 +19,128 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Add the images and tags used by the Operator by setting an environment variable, such as `RELATED_IMAGE_MEMCACHED`, in the `config/manager/manager.yaml` file:
++
+.Example `config/manager/manager.yaml`
+[%collapsible]
+====
+[source,yaml]
+----
+...
+spec:
+  ...
+    spec:
+      ...
+      containers:
+      - command:
+        - /manager
+        ...
+        env:
+        - name: RELATED_IMAGE_MEMCACHED
+          value: "docker.io/memcached@sha256:00b68b00139155817a8b1d69d74865563def06b3af1e6fc79ac541a1b2f6b961"
+----
+====
+
+. Replace hard-coded image references with environment variables in the following files:
+
+* For Go-based Operator projects, update the `controllers/memcached_controller.go` file with the following changes:
++
+.`controllers/memcached_controller.go` diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+// deploymentForMemcached returns a memcached Deployment object
+...
+	Spec: corev1.PodSpec{
+        	Containers: []corev1.Container{{
+-			Image:   "memcached:1.4.36-alpine",
++			Image:   "docker.io/memcached@sha256:00b68b00139155817a8b1d69d74865563def06b3af1e6fc79ac541a1b2f6b961",
```

Comment:
This will pull the image with digest from the environment:

```suggestion
+			Image:   os.Getenv("RELATED_IMAGE_MEMCACHED"),
```

Optionally, you can mention that `os.Getenv` will return an empty string if the variable is unset, so users may want to validate that the env variable is present first.